lib/Db/OldPassword.php

```diff
@@ -26,12 +26,12 @@
 
 /**
  * @method int getId()
- * @method void setId(\int $id)
+ * @method void setId(int $id)
```

Comment:
Please share the error which is fixed through this

Reply:
```
lib/Db/OldPassword.php:28 PhanUndeclaredTypeParameter Parameter $id has undeclared type \int
lib/Db/OldPassword.php:29 PhanParamSignatureMismatch Declaration of function setId(\int $id) : void should be compatible with function setId(int $id) : void defined in ../../lib/public/AppFramework/Db/Entity.php:28
```